### PR TITLE
Replace boot2docker with docker-machine

### PR DESCRIPTION
**Pull request by mbonaci**
Pending landing of the patch for a known VBox 5.0.2 bug

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,12 +51,13 @@ FATA[0000] Post http:///var/run/docker.sock/v1.17/images/create?fromImage=cockro
...
-boot2docker init
-# Start Boot2Docker. This will need to be run once per reboot.
-boot2docker start
-# Setup environment variables. This will need to be run once per shell.
-$(boot2docker shellinit)
+# After installing [Docker Toolbox](https://www.docker.com/toolbox), start the VM (docker host):
+$ docker-machine start default
+# then fire the following command to setup docker environment variables:
+$ eval $(docker-machine env default)
+# The previous command sets "default" as the active docker host VM.
```
Comment (tamird) [inline README.md]: doesn't `docker-machine start default` do this?

Comment (tamird) [inline README.md]: ah, no it doesn't.

Comment (mbonaci) [inline README.md]: Yeah, but if you work with multiple terminal windows you'd need to use this `eval` command in all besides the one you ran `start default` in (if I remember correctly)

Comment (tamird) [inline README.md]: actually you have to do it in all of them; that command is the equivalent of `boot2docker shellinit`

**Comment by tamird** [comment]
Virtualbox 5.0.4 is released. Can you update this to remove other references to boot2docker in the project?

**Comment by mbonaci** [comment]
So Readme is OK, I've just submitted PR for `build/docker-init.sh`. It's important to `source` the docker-init.sh, since regular execution doesn't change the active `docker-machine` to `default`. I've just tested it.

**Pull request merged**